UnitsNet/UnitDefinitions/MolarMass.json

```diff
@@ -0,0 +1,43 @@
+﻿{
+  "Name": "MolarMass",
+  "BaseUnit": "KilogramPerMole",
+  "XmlDoc": "In chemistry, the molar mass M is a physical property defined as the mass of a given substance (chemical element or chemical compound) divided by the amount of substance.",
+  "Units": [
+    {
+      "SingularName": "GramPerMole",
+      "PluralName": "GramsPerMole",
+      "FromUnitToBaseFunc": "x/1e3",
+      "FromBaseToUnitFunc": "x*1e3",
+      "Prefixes": [ "Nano", "Micro", "Milli", "Centi", "Deci", "Deca", "Hecto", "Kilo" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "g/mol" ]
+        },
+        {
+          "Culture": "ru-RU",
+          "Abbreviations": [ "г/моль" ],
+          "AbbreviationsWithPrefixes": [ "нг/моль", "мкг/моль", "мг/моль", "сг/моль", "дг/моль", "даг/моль", "гг/моль", "кг/моль" ]
+        }
+      ]
+    },
+    {
+      "SingularName": "PoundPerMole",
+      "PluralName": "PoundsPerMole",
+      "FromUnitToBaseFunc": "x*0.45359237",
+      "FromBaseToUnitFunc": "x/0.45359237",
+      "Prefixes": [ "Kilo", "Mega" ],
+      "Localization": [
+        {
+          "Culture": "en-US",
+          "Abbreviations": [ "lb/mol" ],
+          "AbbreviationsWithPrefixes": [ "KLbs/mol", "MLbs/mol" ]
```

Comment:
Kilo should be lowercase `klbs`. Also, I think I prefer the singular form when it comes to abbreviations. I know there are varying practices here, but unless `klbs/mol` is clearly preferred over `klb/mol` in this domain then I think we should use singular form to stay consistent in the library.